.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
this is just the default configuration that is auto-created if you don't have one